src/mateacademy/homework/homework_11/StrModifier.java

```diff
@@ -0,0 +1,40 @@
+package mateacademy.homework.homework_11;
+
+import java.util.ArrayDeque;
+import java.util.Iterator;
+import java.util.Scanner;
+
+public class StrModifier {
+
+    private int i;
+    private Integer container;
+    private Scanner scanner;
+    private ArrayDeque arrayDeque;
+
+    public StrModifier() {
```

Comment:
better create one method which will do all work
 try don't use constructor for this purpose